entrypoint.sh

```diff
@@ -467,9 +485,13 @@ function restart_driver() {
 
     ${load_pci_hyperv_intf} && exec_cmd "modprobe -d /host pci-hyperv-intf"
 
+    timestamp_print "Apply blacklisted mofed modules file to host (${OFED_BLACKLIST_MODULES_FILE})"
+    generate_ofed_modules_blacklist 
     ${UNLOAD_STORAGE_MODULES} && unload_storage_modules
 
     exec_cmd "/etc/init.d/openibd restart"
+    timestamp_print "Remove blacklisted mofed modules file from host"
```

Comment:
Based on [1] and [2] this code won't be executed if openibd restart fails

[1] https://github.com/Mellanox/doca-driver-build/blob/69d08d646db208b5e1e16b489f831b0d239986fe/entrypoint.sh#L153
[2] https://github.com/Mellanox/doca-driver-build/blob/69d08d646db208b5e1e16b489f831b0d239986fe/entrypoint.sh#L62

Reply:
Fixed by using trap